data/helpdata.txt

```diff
@@ -351,176 +352,176 @@ categories = "Bonus", "Nuisance"
+in the destination city. The initial revenue depends on the trade already produced by the two cities \
+involved and their distance apart. The destination civilization learns about the location of the origin \
+city, if it is not already known.\
 "), _("\
-The ongoing Trade Route that is established benefits both its origin and destination Cities equally by \
-generating Trade points for each City every turn. The amount of ongoing Trade increases with the size of \
-each City and the distance between them, and may also depend on the factors mentioned above. The trade \
-relationship also gives each Civilization limited ongoing intelligence about the partner City. If a Trade \
-Route ever becomes unsustainable, for instance, if War breaks out in a ruleset where enemies cannot Trade, \
+The ongoing trade route that is established benefits both its origin and destination cities equally by \
+generating trade points for each city every turn. The amount of ongoing trade increases with the size of \
+each city and the distance between them, and may also depend on the factors mentioned above. The trade \
+relationship also gives each civilization limited ongoing intelligence about the partner city. If a trade \
+route ever becomes unsustainable, for instance, if War breaks out in a ruleset where enemies cannot trade, \
 it may be permanently canceled.\
 "), _("\
-Each City can only support a limited number of Trade Routes (the exact limit is set by ruleset and a \
-server setting). If you attempt to establish more, the Trade Route with the smallest ongoing revenue is \
-canceled if it would be less than the new Trade Route. Otherwise, the origin Civilization can still gain \
-initial revenue by entering the marketplace and selling Trade goods, but it is reduced to a third.\
+Each city can only support a limited number of trade routes (the exact limit is set by ruleset and a \
+server setting). If you attempt to establish more, the trade route with the smallest ongoing revenue is \
+canceled if it would be less than the new trade route. Otherwise, the origin civilization can still gain \
+initial revenue by entering the marketplace and selling trade goods, but it is reduced to a third.\
 "), _("\
-This limit is enforced only when attempting to establish a new Trade Route. If you temporarily lose the \
-ability to establish as many Trade Routes as a City already has, existing ones are not affected.\
+This limit is enforced only when attempting to establish a new trade route. If you temporarily lose the \
+ability to establish as many trade routes as a city already has, existing ones are not affected.\
 "), _("\
-Beware: in rulesets where Plague is enabled, it can travel along Trade Routes. See the section on Plague for \
-more details.\
+Beware: in rulesets where Plague is enabled, it can travel along trade routes. See the page on Plague in the \
```

Comment:
Minor: I'd write "the Plague page" or "the page on plague" (the first referring to the title, the second to the event)